src/createPackage/models/createPackageManager.ts

```diff
@@ -338,6 +343,81 @@ export class CreatePackageManager {
     await fsPromise.writeFile(metadataFilePath, recordMetadata);
   }
 
+  public async createExportJsonMetadata(job: JobExportResponse): Promise<void> {
+    this.logger.info(`Creating metadata.json file for gpkg in path "${job.parameters.relativeDirectoryPath}" for jobId ${job.id}`);
+    const record = await this.rasterCatalogManager.findLayer(job.internalId as string);
+    const featuresRecords = parseFeatureCollection(job.parameters.roi);
+
+    const maxResolutionDeg = Math.max(
```

Comment:
consider to to refactor to a method named -> getExportedPackageMaxResolution, for a more readable code, to be called once with resolutionDegree parameters and once with resolutionMeters parameters

Reply:
not relevant - only line calculation each time - better the implicity